python3: can't open file 'run_mnist.py': [Errno 2] No such file or directory
